scrape/scrape_test.go

```diff
@@ -3154,3 +3155,163 @@ func TestTargetScrapeIntervalAndTimeoutRelabel(t *testing.T) {
 	require.Equal(t, "3s", sp.ActiveTargets()[0].labels.Get(model.ScrapeIntervalLabel))
 	require.Equal(t, "750ms", sp.ActiveTargets()[0].labels.Get(model.ScrapeTimeoutLabel))
 }
+
+func TestTargetScrapeConfigWithLabels(t *testing.T) {
```

Comment:
Do you mind doing this in the `cases := struct{...}` format? Just makes it easier to read.